snow/consensus/snowball/flats_test.go

```diff
@@ -10,9 +10,10 @@ import (
 
 var _ Consensus = (*Flat)(nil)
 
-func NewFlat(factory Factory, params Parameters, choice ids.ID) Consensus {
+func NewFlat(_ Factory, params Parameters, choice ids.ID) Consensus {
```

Comment:
Could we either honor the param and keep passing in `SnowballFactory` in the one location it's used or remove it and this will just be a `NewSnowballFlat` function?

I'd lean towards honoring the param